Dockerfile

```diff
@@ -2,17 +2,28 @@ FROM centos:7
 MAINTAINER Federico Simoncelli <fsimonce@redhat.com>
 
 RUN yum update -y && \
-    yum install -y golang openscap-scanner && \
+    yum install -y \
+    openscap-scanner && \
     yum clean all
 
+    #needs go 1.7 for context
+RUN cd /tmp && \
+    curl -LO https://storage.googleapis.com/golang/go1.7.linux-amd64.tar.gz && \
+    tar -C /usr/local -xvzf go1.7.linux-amd64.tar.gz && \
+    rm -f go1.7.linux-amd64.tar.gz
```

Comment:
that is fine, but I don't think you need it for the runtime right? just to build the image-inspector and after that point it is just a garbage in the image ;)